pkg/httpclient/scoped_client_test.go

```
@@ -252,18 +337,20 @@ func TestCheckServerStatusWith4xx(t *testing.T) {
 
 // Helpers below
 func newTestRequest() (client Client, context *gin.Context, recorder *httptest.ResponseRecorder) {
+	_ = InitClientWithDefaults()
```

`_ = InitClientWithDefaults()` -> `InitClientWithDefaults()`

---

I like this more as it explicitly shows that code ignores the error